intezer_sdk/results.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Dict, Tuple
+
+from intezer_sdk.api import IntezerApi, raise_for_status
+
+
+class Results:
+    def __init__(self, url: str, api: IntezerApi, data: Dict):
+        """
+        Store list of data into pages.
+        :param url: Url to request new data from.
+        :param api: Istance of publice intezer api for request server.
+        :param data: filtered data.
+        """
+        self.api = api
+        self.data = data
+        self.pages = []
+        self._current_page = None
+        self._url = url
+        self._page_size = data["limit"]
+        self._current_page_number = 0
+        self.total_count = 0
+        self._current_offset = 0
+
+    def __iter__(self):
+        """Iterate for page."""
+        if not self._current_page:
+            self._fetch_page()
+        yield from self._current_page
+        self._fetch_page()
+        yield from iter(self)
+
+    def current_page(self) -> List:
```

Comment:
let's add type hint in the return (in all the PR)

Reply:
Not sure I understand what you mean.